docs/reference/query-languages/esql/esql-commands.md

```diff
@@ -818,6 +820,15 @@ FROM employees
 | RENAME first_name AS fn, last_name AS ln
 ```
 
+With multiple `RENAME` commands:
+
+```esql
+FROM employees
```

Comment:
This is not the same as the version in docs.csv-spec:104

Reply:
I will be automating the inclusion of examples soon, so my PR will fix this, making it the same as the example in the csv-spec. If you prefer this two-line RENAME, then edit the example in the csv-spec

Reply:
Wait, I see now, this is the third example, and it does not exist at all in rename.csv-spec

Reply:
Added the example